package.json

```diff
@@ -16,45 +16,37 @@
   },
   "module": "dist/videojs-dock.es.js",
   "generator-videojs-plugin": {
-    "version": "5.0.0-7"
+    "version": "7.0.2"
   },
   "scripts": {
     "prebuild": "npm run clean",
     "build": "npm-run-all -p build:*",
-    "build:css": "npm-run-all build:css:sass build:css:bannerize",
-    "build:css:bannerize": "bannerize dist/videojs-dock.css --banner=scripts/banner.ejs",
-    "build:css:sass": "node-sass src/plugin.scss dist/videojs-dock.css --output-style=compressed --linefeed=lf",
-    "build:js": "npm-run-all build:js:rollup-modules build:js:rollup-umd build:js:bannerize build:js:uglify",
-    "build:js:bannerize": "bannerize dist/videojs-dock.js --banner=scripts/banner.ejs",
-    "build:js:rollup-modules": "rollup -c scripts/modules.rollup.config.js",
-    "build:js:rollup-umd": "rollup -c scripts/umd.rollup.config.js",
-    "build:js:uglify": "uglifyjs dist/videojs-dock.js --comments --mangle --compress --ie8 -o dist/videojs-dock.min.js",
-    "build:test": "rollup -c scripts/test.rollup.config.js",
-    "clean": "rimraf dist test/dist",
-    "postclean": "mkdirp dist test/dist",
+    "build:css": "node-sass src/plugin.scss dist/videojs-dock.css --output-style=compressed --linefeed=lf",
```

Comment:
I noticed the same thing in videojs-contextmenu-ui. Is the idea here that we can remove Sass in a subsequent PR to avoid source changes in this one? That should probably happen before another release is done.

Reply:
Question was answered offline in the affirmative.